Core/Core/Kits/ISpeckleConverter.cs

```diff
@@ -57,6 +57,14 @@ public interface ISpeckleConverter
     /// <returns></returns>
     public List<object> ConvertToNative(List<Base> objects);
 
+    /// <summary>
+    /// Connects a list of Speckle objects between each other.
+    /// </summary>
+    /// <param name="obj"></param>
+    /// <param name="objects"></param>
+    /// <returns></returns>
+    public void ConnectElement(ApplicationPlaceholderObject obj, List<ApplicationPlaceholderObject> objects);
```

Comment:
Reconsider adding a new binding to converters, as this would require *all* of our converters to implement this